datafusion/src/physical_plan/file_format/parquet.rs

```diff
@@ -217,6 +217,7 @@ impl ExecutionPlan for ParquetExec {
 
         let file_schema_ref = self.base_config().file_schema.clone();
         let join_handle = task::spawn_blocking(move || {
+            let f = partition.clone();
```

Comment:
perhaps we can pass partition as a reference to `read_partition` to avoid the clone here? I noticed read_partition is already perfoming clone inside.

Reply:
fixed